content/logs/faq/log-collection-troubleshooting-guide.md

```diff
@@ -142,6 +142,32 @@ config_providers:
 
 When using Journald in a containerized environment, make sure to follow the instructions in the [journald integration][4] as there is a specific file used to mount to the Agent.
 
+## Serverless environment
+
+### Logs from lambda functions are not visible in log explorer
+
+See the [Datadog-AWS Log integration][9] to configure your environment.
+
+If you still do not see your logs, double-check the following points:
+
+#### are logs expectedly dropped? 
+
+Check if logs appear in the [Datadog Live Tail][10]. Source is should to be `aws` or `cloudwatch` (see the `parse_event_source()` function in the [Datadog Python lambda function][11] for details).
+
+If they appear in the Live Tail page, you probably explicitly excluded those logs from indexes. 
+
+If they do not appear in the Live Tail page, remind that we drop logs older than 6 hours.
+
+#### configuration of datadog lambda function
+
+Check Datadog lambda configuration parameter:
+
+* `<API_KEY>` : Should be set with your [Datadog API key][12] either directly in the Python code, or alternatively as a environment variable.
```

Comment:
Let's encourage them to double check that the apikey is the same as in their platform (and provide a link to the apikey page)